sunpy/io/cdf.py

```diff
@@ -89,7 +89,12 @@ def read_cdf(fname):
                           'Assigning dimensionless units.')
                 unit = u.dimensionless_unscaled
 
-            if data.ndim == 2:
+            if data.ndim > 2:
+                # Skip data with dimensions >= 3 and give user warning
+                warn_user(f'The variable "{var_key}" has more than 2 dimensions, '
+                          'which is not supported at the moment. '
+                          f'Skipping variable "{var_key}" for now.')
```

Comment:
```suggestion
                warn_user(f'The variable "{var_key}" has more than 2 dimensions, which is unsupported .')
```

Maybe? I feel like the message could be shorter but maybe that is too short.

Reply:
Yes, I mentioned in my reply to the issue (#5869) that the message might need some fine-tuning (not sure which is the better place for discussion). I'm open for suggestions. 🙂

Reply:
Never say never! :smile: But it's fine for me. I only would suggest to explicitly state that the corresponding variable will be skipped, so that this is clear for the user. For example:

```suggestion
                warn_user(f'The variable "{var_key}" has been skipped because it has more than 2 dimensions, which is unsupported. ')
```
```

Reply:
That works for me.